releasenotes-dca/notes/Add-pod-tolerations-metric-to-KSM-core-check-1cc56a7c1986dd83.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Adding kube_pod_tolerations metric to KSM core check
```

Comment:
Should we mention `kubernetes_state.pod.tolerations` which is what the user sees instead?

Reply:
Yes makes sense, thanks for pointing this out.

I updated the changes to reflect the metric name as exposed in the app.